### PR TITLE
i18n(pt-BR): Updated `/reference/api-reference.mdx`

### PR DESCRIPTION
**Pull request by Alynva**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
  - Updated `/reference/api-reference.mdx` continuing https://github.com/withastro/docs/pull/3899 based on commits in the original pages (see the commit history for more references)

### CODE REVIEW

File: src/content/docs/pt-br/reference/api-reference.mdx [hunk 1]
```diff
@@ -749,58 +875,132 @@ const postagensRascunhoBlog = await getCollection('blog', ({ data }) => {
...
+---
+import { getEntry } from 'astro:content';
+
+// Obtém `src/content/blog/empreendimento.md`
+const empreendimentoPostagem = await getEntry('blog', 'empreendimento');
+
+// Obtém `src/content/capitaes/picard.yaml`
+const picardPerfil = await getEntry('capitaes', 'picard');
+
+// Obtém o erfil referenciado por `data.capitao`
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/reference/api-reference.mdx:902]: Erro de digitação
```suggestion
// Obtém o perfil referenciado por `data.capitao`
```

File: src/content/docs/pt-br/reference/api-reference.mdx [hunk 2]
```diff
@@ -749,58 +875,132 @@ const postagensRascunhoBlog = await getCollection('blog', ({ data }) => {
...
+const empreendimentoCapitaoPerfil = await getEntry(empreendimento.data.capitao);
+---
+```
+
+Veja o guia `Coleções de Conteúdo` para exemplos de  [consultando entradas de coleção](/pt-br/guides/content-collections/#querying-collections).
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/reference/api-reference.mdx:907]:
```suggestion
Veja o guia `Coleções de Conteúdo` para exemplos de como [consultar entradas de coleção](/pt-br/guides/content-collections/#querying-collections).
```

File: src/content/docs/pt-br/reference/api-reference.mdx [hunk 3]
```diff
@@ -749,58 +875,132 @@ const postagensRascunhoBlog = await getCollection('blog', ({ data }) => {
...
 
-#### `slug`
+#### `collection`
 
-**Tipo de Exemplo:** `'entrada-1' | 'entrada-2' | ...`
+**Disponível para:** coleções `type: 'content'` e `type: 'data'`  
+**Tipo de Exemplo:** `'blog' | 'autores' | ...`
 
-Uma slug pronta para URLs. Por padrão é o `id` sem a extensão de arquivo, mas pode ser sobreescrita ao definir [a propriedade `slug`](/pt-br/guides/content-collections/#defining-custom-slugs) no frontmatter de um arquivo.
+O nome de uma pasta de nível superior dentro de `src/content/` onde as entradas são localizadas. Este é o nome usado para referencias a coleção no seu esquema e em funções de consulta.
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/reference/api-reference.mdx:978]:
```suggestion
O nome de uma pasta de nível superior dentro de `src/content/` onde as entradas são localizadas. Este é o nome usado para referenciar a coleção no seu esquema e em funções de consulta.
```

**Review by Yan-Thomas — APPROVED** [review]
Thanks @Alynva, a few `api-reference.mdx` suggestions here. For the other pages, my review is in the other PR.

**Review by Yan-Thomas — CHANGES_REQUESTED** [review]
oops, approved by accident

**Comment by Alynva** [comment]
@Yan-Thomas I'll mark this as draft as this PR includes commits from https://github.com/withastro/docs/pull/3899

**Comment by Yan-Thomas** [comment]
@Alynva can you take a look and fix the conflicts here? I suppose removing the changes to the files from the other PR would solve it

**Comment by Alynva** [comment]
@Yan-Thomas I think that everything is right now

**Comment by Alynva** [comment]
Ok, its fixed now

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @Alynva 🙌

**Pull request merged**